script/brave_license_helper.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import os
+
+
+def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
+    # Exclude these specific paths from needing a README.chromium file.
+    prune_paths.update([
+        # Formerly external Brave code which has moved to brave-core
+        # (i.e these are already covered by the Brave Browser license notice).
+        os.path.join('brave', 'vendor', 'bat-native-ads'),
+        os.path.join('brave', 'vendor', 'bat-native-confirmations'),
+        os.path.join('brave', 'vendor', 'bat-native-ledger'),
+        os.path.join('brave', 'vendor', 'brave-ios'),
+        os.path.join('brave', 'vendor', 'brave_base'),
+
+        # Brave overrides to third-party code, also covered by main notice.
+        os.path.join('brave', 'third_party', 'android_deps'),
+        os.path.join('brave', 'third_party', 'blink'),
+        os.path.join('brave', 'third_party', 'libaddressinput'),
+    ])
+
+    # Add the licensing info that would normally be in a README.chromium file.
+    # This is for when we pull in external repos directly and therefore can't
```

Comment:
this comment doesn't actually apply to this list because we own all these repos and could add the README.chromium if we wanted. I'm fine with leaving this list as opposed to updating all of them which would be a big pita, but just want to update the comment here

Reply:
I thought it wouldn't make a lot of sense to have a `README.chromium` in an external repository, but if we are happy to have that, then I can take care of that in a follow-up.

This comment however is correct. We don't control `boto` or `sparkle` for example.

Reply:
I will take "and therefore can't add an extra README.chromium file." out of the comment to make this clearer.